test/message/test_runner_output.js

```diff
@@ -328,3 +328,13 @@ test('subtest sync throw fails', async (t) => {
     throw new Error('thrown from subtest sync throw fails at second');
   });
 });
+
```

Comment:
Can you add a new test or update an existing one that explicitly sets a large timeout and the test does not time out.